test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -25,81 +31,162 @@ describe('IexecAccessors', async () => {
+        expect(task.consensusValue).to.equal(HashZero);
+        expect(task.revealCounter).to.equal(0);
+        expect(task.winnerCounter).to.equal(0);
+        expect(task.contributors.length).to.equal(0);
+        expect(task.resultDigest).to.equal(HashZero);
+        expect(task.results).to.equal('0x');
+        expect(task.resultsTimestamp).to.equal(0);
+        expect(task.resultsCallback).to.equal('0x');
+    });
+
     it('countCategory', async function () {
         expect(await iexecPocoAsAnyone.countCategory()).to.equal(5);
     });
+
     it('appRegistry', async function () {
         expect(await iexecPocoAsAnyone.appregistry()).to.equal(
             (await deployments.get('AppRegistry')).address,
         );
     });
+
     it('datasetRegistry', async function () {
         expect(await iexecPocoAsAnyone.datasetregistry()).to.equal(
             (await deployments.get('DatasetRegistry')).address,
         );
     });
+
     it('workerpoolRegistry', async function () {
         expect(await iexecPocoAsAnyone.workerpoolregistry()).to.equal(
             (await deployments.get('WorkerpoolRegistry')).address,
         );
     });
+
     it('teeBroker', async function () {
         expect(await iexecPocoAsAnyone.teebroker()).to.equal(ethers.constants.AddressZero);
     });
+
     it('callbackGas', async function () {
         expect(await iexecPocoAsAnyone.callbackgas()).to.equal(100_000n);
     });
+
     it('contributionDeadlineRatio', async function () {
         expect(await iexecPocoAsAnyone.contribution_deadline_ratio()).to.equal(7);
     });
+
     it('revealDeadlineRatio', async function () {
         expect(await iexecPocoAsAnyone.reveal_deadline_ratio()).to.equal(2n);
     });
+
     it('finalDeadlineRatio', async function () {
         expect(await iexecPocoAsAnyone.final_deadline_ratio()).to.equal(10n);
     });
+
     it('workerpoolStakeRatio', async function () {
         expect(await iexecPocoAsAnyone.workerpool_stake_ratio()).to.equal(30n);
     });
+
     it('kittyRatio', async function () {
         expect(await iexecPocoAsAnyone.kitty_ratio()).to.equal(10n);
     });
+
     it('kittyMin', async function () {
         expect(await iexecPocoAsAnyone.kitty_min()).to.equal(1_000_000_000n);
     });
+
     it('kittyAddress', async function () {
         expect(await iexecPocoAsAnyone.kitty_address()).to.equal(
             '0x99c2268479b93fDe36232351229815DF80837e23',
         );
     });
+
     it('groupMemberPurpose', async function () {
         expect(await iexecPocoAsAnyone.groupmember_purpose()).to.equal(4n);
     });
+
     it('eip712domainSeparator', async function () {
         expect(await iexecPocoAsAnyone.eip712domain_separator()).to.equal(
             '0xfc2178d8b8300e657cb9f8b5a4d1957174cf1392e294f3575b82a9cea1da1c4b',
         );
     });
 });
+
+/**
+ * Build orders, create a deal, and initialize to create a task.
+ */
+async function createTask() {
+    const appPrice = 1000;
```

Comment:
I extracted this in a separate function because (1) I find the test more readable and (2) in preparation for other tests (`viewContribution()` for e.g.) where we also need to create a task and contribute.